patches/chrome-browser-net-system_network_context_manager.cc.patch

```diff
@@ -11,3 +11,12 @@ index 2163341048bbe4dbc669bb4fa26637fde3c9d90c..edd9e1882fb3858ad30cf96c2fd3555f
          std::move(client), traffic_annotation);
    }
  
+@@ -569,6 +569,8 @@ void SystemNetworkContextManager::ConfigureDefaultNetworkContextParams(
+         content::BuildOSCpuInfo(content::IncludeAndroidBuildNumber::Exclude,
+                                 content::IncludeAndroidModel::Include));
+   }
++  //
```

Comment:
extra line here, but I think we can get rid of this patch anyway as discussed in dm